index.html

```diff
@@ -1,8 +1,153 @@
+      <figure>
+         <img src="IMAGES\KALLIE1.JPG" width="400" height="300">
+         <figcaption>Kallie, at the shelter.</figcaption>
+      </figure>
+
+      <figure>      
+         <img src="images\KALLIE2.JPG" width="400" height="300">
+         <figcaption>Kallie, a happy housecat.</figcaption>
+      </figure>
+      </article>
+
+
+      <br><center><br><a href = #top>Back to Top</a></center>
+   
+
+      <article id = "pebbles">
+      <h2>Pebbles</h2>
+      <p class = "funParagraph" id="testP">Ah, Spring 2020. My wife and I were bored and in quarantine when we thought, "hey, since this thing may go on for a couple months, we should get a kitten!" As luck would have it, a friend of my sister's had a barn cat who had just delivered a litter of kittens who were in need of homes! My wife and I went to the farm and, guess what kind of kittens they had?? That's right, calico! Now, our reasoning behind getting a calico kitten this time was because they are more than likely female and we didn't want a male cat spraying everywhere. My wife first picked up a scared, timid little kitten and I (sometimes regretfully), said we should get the one that was a little more active. We brought the kitten home a couple weeks later and initially named her Tequila (Tiki for short). The name didn't stick. We then thought about Pickles, which lead us to Pebbles, and the name stuck. Pebbles is affectionately refered to as "Crackhead". Her hobbies include scaling walls, opening cat-proof containers to get food, helping me with coding, and bullying <a href="#kallie">Kallie</a> out of her food.</p>
+      <figure>
+         <img src="images\PEBBLES1.JPG" width="400" height="300">
+         <figcaption>Pebbles, the adventurer.</figcaption>
+      </figure>
+
+      <figure>
+         <img src="images\PEBBLES2.JPG"width="400" height="300">
+         <figcaption>Pebbles, the coding kitty.</figcaption>
+      </figure>
+      </article>
+      
+      <center><br><a href = #top>Back to Top</a></center><br>
+          
+      </main>
+
+      <main>
+      <h1>Dogs</h1>
+
+      <article id = "bud">
+      <h2>Bud</h2>
+      <p>Bud (legal name, Budweiser&reg;) was technically not my dog but he was an integral part of my childhood. Bud belonged to my sister; he was a birthday present from her husband after Bud had been found abandoned near their house. Bud absolutely <b>loved</b> beer. Specifically, he loved Budweiser beer - hence, his name. When my sister divorced, Bud came to live with my parents and me (plus <a href=#callieholly>Callie &amp; Holly</a>). Bud and the cats tolerated each other well enough. My family thoroughly believes, to this day, that Bud was part human. Bud also loved popsicles in the summer, snuggling behind my mom on her favorite chair, and taking naps with me on the couch. Much to my dismay, when my sister and her husband remarried, Bud went back to live with them and his rottweiler brother, Yukon.</p>
```

Comment:
When it comes to paragraphs like this, for readability, I recommend hitting enter after every sentence or so, so you don't have such a long single line where you need to scroll over so much. Don't worry, this won't change how it appears on your webpage, but just makes the code a little more readable!